examples/todo/src/Data/List/Utils.purs

```diff
@@ -0,0 +1,70 @@
+module Data.List.Utils (
```

Comment:
There doesn't seem to be a library of common list manipulators in a library that's easy to get at. They seem to be scattered all over the place. Some of them are simple synonyms, like `join` & `intercalate` in Data.Foldable, but definitely not anything as extensive and as centralised as Data.List.Utils from Haskell's MissingH.